plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -1121,7 +1047,7 @@ def add_traces(
         # Convert the data properties
         if isinstance(max_n_samples, (int, np.integer)) or max_n_samples is None:
             max_n_samples = [max_n_samples] * len(data)
-        if isinstance(downsamplers, AbstractSeriesAggregator) or downsamplers is None:
+        if isinstance(downsamplers, DataPointSelector) or downsamplers is None:
```

Comment:
Shouldn't this be `AbstractAggregator`? :thinking: 
Or do we only need to cover data point selection downsamplers here?